From 2d6fe1cb7a1c451fcdc81d1bb862cf5cd592d2d5 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 13 Jan 2021 19:21:29 +0800
Subject: [PATCH] arm64: rockchip_defconfig: set CONFIG_BLK_DEV_LOOP_MIN_COUNT
 to 16

Sync to gki_defconfig

CONFIG_BLK_DEV_LOOP_MIN_COUNT must be at least 16 in order
to pass VTS.

Signed-off-by: Tao Huang <huangtao@rock-chips.com>
Change-Id: I15e46ac79dc6f149914a540e311369177bccdc23
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 7c3048c4d3f5..e02a97a13389 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -263,6 +263,7 @@ CONFIG_MALI_MEMORY_GROUP_MANAGER=y
 CONFIG_DMA_CMA=y
 CONFIG_ZRAM=y
 CONFIG_BLK_DEV_LOOP=y
+CONFIG_BLK_DEV_LOOP_MIN_COUNT=16
 CONFIG_BLK_DEV_RAM=y
 CONFIG_BLK_DEV_RAM_SIZE=8192
 CONFIG_BLK_DEV_NVME=y
-- 
2.35.3

